lib/exception_hunter/data_redacter.rb

```diff
@@ -0,0 +1,22 @@
+module ExceptionHunter
+  class DataRedacter
+    class << self
```

Comment:
Maybe convert to instance methods? I know it's not necessary to have state but it can cause issues in the future